etcdserver/api/v3rpc/watch.go

```diff
@@ -17,17 +17,19 @@ package v3rpc
 import (
 	"io"
 
+	"github.com/coreos/etcd/etcdserver"
 	pb "github.com/coreos/etcd/etcdserver/etcdserverpb"
 	"github.com/coreos/etcd/storage"
 	"github.com/coreos/etcd/storage/storagepb"
 )
 
 type watchServer struct {
+	server    *etcdserver.EtcdServer
```

Comment:
can we use etcdserver.Server and Cluster interface rather than the implementation?